src/test/FuzzerImpl.cpp

```diff
@@ -1421,6 +1422,7 @@ TransactionFuzzer::initialize()
     resetRandomSeed(1);
     mApp = createTestApplication(mClock, getFuzzConfig(0));
     OrderBookIsNotCrossed::registerAndEnableInvariant(*mApp);
+    ConstantProductInvariant::registerAndEnableInvariant(*mApp);
```

Comment:
Should the invariant be registered instead in `ApplicationImpl::initialize`? (with an ifdef) that way we also check this invariant when running regular tests, not just when fuzzing.

Reply:
or even without the ifdef, should this invariant be usable all the time?

Reply:
Yeah we can just make this a normal invariant.